app/models/gids_redis.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
```

Comment:
I don't think these `require`s are necessary

Reply:
rspec tests fail locally with UninitializedConstant errors

Reply:
when going to http://localhost:3000/v0/gi/calculator_constants after removing requires I get 

```
| 2019-12-18 21:19:41.345846 E [12:puma threadpool 001 sentry_logging.rb:58] [[]] Rails -- uninitialized constant Common::CacheAside.
21:19:41 web.1       | 2019-12-18 21:19:41.346031 E [12:puma threadpool 001 sentry_logging.rb:58] [[]] Rails -- /srv/vets-api/src/app/models/gids_redis.rb:5:in `<class:GIDSRedis>'
21:19:41 web.1       | /srv/vets-api/src/app/models/gids_redis.rb:4:in `<top (required)>'
21:19:41 web.1       | /usr/local/bundle/gems/activesupport-5.2.3/lib/active_support/dependencies/interlock.rb:14:in `block in loading'
21:19:41 web.1       | /usr/local/bundle/gems/activesupport-5.2.3/lib/active_support/concurrency/share_lock.rb:151:in `exclusive'
21:19:41 web.1       | /usr/local/bundle/gems/activesupport-5.2.3/lib/active_support/dependencies/interlock.rb:13:in `loading'
21:19:41 web.1       | /srv/vets-api/src/app/controllers/gids_controller.rb:9:in `service'
21:19:41 web.1       | /srv/vets-api/src/app/controllers/v0/gids/calculator_constants_controller.rb:7:in `index'```

Reply:
Ah, yes, looks like `cache_aside`'s namespacing doesn't match it's path for autoloading.
I think you can remove `require 'common/models/redis_store'`, but will have to keep `require 'common/models/concerns/cache_aside'`.  Seems to be working for me.

Reply:
removed